Bearded.Utilities.Tests/Geometry/CircularArc2Tests.cs

```diff
@@ -10,354 +10,332 @@ namespace Bearded.Utilities.Tests.Geometry
 {
     public sealed class CircularArc2Tests
     {
-        public sealed class ShortArcBetweenDirections
+        private const float epsilon = 0.001f;
+
+        [Property(Arbitrary = new[] { typeof(DirectionGenerators.All) })]
+        public void CreatingShortArcBetweenIdenticalDirections_ReturnsZeroLengthArc(Direction2 fromTo)
```

Comment:
We could rename these into 'facts' like `ShortArcBetweenIdenticalDirectionsIsZeroLength`, which is shorter too (note how I changed "ReturnsZeroLengthArc" to "IsZeroLength"). With that, I don't think the names are too long either, they need that info to be clear.

Reply:
Thanks, I like that suggestion. Applied as best as possible throughout all tests.